### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by whitneykidd**
Add tests to favorite_spec
Update apps#unapprove method
Made "On hold for Applicant Name" a link to the app#show page

### CODE REVIEW

File: app/assets/stylesheets/application.css [hunk 1]
```diff
@@ -70,7 +70,7 @@ img {
 }
 
 .pet_img {
-  width: 70%!important;
+  width: 400px!important;
```
Comment (whitneykidd) [inline app/assets/stylesheets/application.css:73]: I'm still playing around with design here, but this seems to make the pet#show page a bit cleaner

File: app/assets/stylesheets/application.css [hunk 2]
```diff
@@ -81,7 +81,7 @@ h1 {
 
 input {
   font-size: 16px;
-  width: 80%;
+  width: auto;
```
Comment (whitneykidd) [inline app/assets/stylesheets/application.css:84]: This made the 'favorite pet' buttons smaller

File: app/controllers/apps_controller.rb [hunk 3]
```diff
@@ -74,21 +49,12 @@ def update
 
   def unapprove
     @app = Application.find(params[:id])
-    pet_app = PetApplication.where(application_id: @app.id).first #need to change .first if we do single(or multi)
-    # require "pry"; binding.pry
-    pet = Pet.where(id: pet_app.pet_id)
+    pet = Pet.find(params[:pet_id])
+    pet_app = PetApplication.find_by(pet_id: params[:pet_id], application_id: params[:id])
```
Comment (whitneykidd) [inline app/controllers/apps_controller.rb:53]: This pretty much follows the same pattern as app#update now

File: app/views/apps/show.html.erb [hunk 4]
```diff
@@ -7,15 +8,12 @@
...
-
-<h1>Approve Application</h1>
-
 <% @app.pets.each do |pet| %>
       <% if pet.on_hold_for == nil %>
         <%= button_to "Approve Application for #{pet.name}", "/applications/#{@app.id}/update", method: :patch, params: {pet_id: pet.id} %>
       <% else %>
        <% if pet.on_hold_for == @app.name %>
-        <%= button_to "Unapprove Application for #{pet.name}", "/applications/#{@app.id}/unapprove", method: :patch %>
+        <%= button_to "Unapprove Application for #{pet.name}", "/applications/#{@app.id}/unapprove", method: :patch, params: {pet_id: pet.id} %>
```
Comment (janegreene) [inline app/views/apps/show.html.erb:16]: Good catch, thank you!

Comment (whitneykidd) [inline app/views/apps/show.html.erb:16]: Added the params to the button here - same as app#update

File: app/views/pets/show.html.erb [hunk 5]
```diff
@@ -1,27 +1,26 @@
-<div class="container">
-<b>On hold for: <%= @pet.on_hold_for%> </b><br/>
-
+<div class="container"><br/>
```
Comment (whitneykidd) [inline app/views/pets/show.html.erb:1]: Now this div should only show up if a pet has an approved app

File: app/views/pets/show.html.erb [hunk 6]
```diff
@@ -1,27 +1,26 @@
...
 
 <div class="column-<%= @pet.id %>"><p><b>Description:</b> <%= @pet.description %></p>
 <p><b>Age:</b> <%= @pet.age %></p>
 <p><b>Sex:</b> <%= @pet.sex %></p>
 <p><b>Adoption Status:</b> <%= @pet.status %></p>
 
 <%= link_to "Update Pet", "/pets/#{@pet.id}/edit" %> |
-<%= link_to "Delete Pet", "/pets/#{@pet.id}", method: :delete, data: {confirm: "Confirm Pet Deletion"} %><br/><br/>
+<%= link_to "Delete Pet", "/pets/#{@pet.id}", method: :delete, data: {confirm: "Confirm Pet Deletion"} %> |
+<%= link_to "View all Applications", "/pets/#{@pet.id}/applications" %> |
```
Comment (whitneykidd) [inline app/views/pets/show.html.erb:19]: Styling update

File: spec/features/favorite/index_spec.rb [hunk 7]
```diff
@@ -34,6 +34,11 @@
                         status: "Adoptable")
     end
 
+    after(:each) do
```
Comment (janegreene) [inline spec/features/favorite/index_spec.rb:37]: this is handy to know

**Pull request merged**